pkgs/applications/misc/watt-toolkit/default.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, stdenv
+, fetchurl
+, dpkg
+, autoPatchelfHook
+, makeWrapper
+, copyDesktopItems
+, makeDesktopItem
+, libcap
+, zlib
+, libcxx
+, fontconfig
+, icu
+, openssl
+, libX11
+}:
+
+stdenv.mkDerivation rec {
+  pname = "watt-toolkit";
+  version = "2.8.4";
+
+  src = fetchurl {
+    url = "https://github.com/BeyondDimension/SteamTools/releases/download/${version}/Steam++_linux_x64_v${version}.deb";
+    sha256 = "sha256-s44C8RKmj9TtmG37O2mLrut+JWkHnA//DoNANelCBHs=";
+  };
+
+  icon = fetchurl {
+    url = "https://github.com/BeyondDimension/SteamTools/raw/${version}/resources/AppIcon/Logo_64.png";
+    sha256 = "sha256-HH4rFntrRXxoXQ1IkBR+33DrKj7u+xKqQt3QxyGy9J0=";
+  };
+
+  buildInputs = [
+    libcap
+    zlib
+    libcxx
+    fontconfig
+    icu
+    openssl
+    libX11
+  ];
+  nativeBuildInputs = [ autoPatchelfHook makeWrapper copyDesktopItems ];
+
+  dontConfigure = true;
```

Comment:
If there is no makefile or autoconf file which would trigger we don't need to disable this.